src/helicopter.jl

```diff
@@ -0,0 +1,227 @@
+    #=   
+    #### NOTE ON STATES::                                                   
+    x[14:16] --> wind velocities (estimated or ground-truth) 
+    x[14:16] --> wind "accelerations" 
+
+    #### NOTE ON DYNAMICS:
+    ASSUME: No blade-flapping dynamics;
+            No horizontal+Vertical fins;
+            No stabilizer;
+            Centre of Pressure == C.O.G
+
+    #### NOTE ON CONTROL INPUTS:
+    u[1] = Main Rotor tilt angle with vertical in Helicopter's X-Z plane (*check!) 
+    u[2] = Main Rotor tilt angle with vertical in Helicopter's Y-Z plane
+    u[3] = Main Rotor Thrust
+    u[4] = Tail Rotor Thrust
+    =#
+    ####################################################################
+    r = x[1:3] #x y z
+    q = x[4:7] # quaternion
+    v = x[8:10] #u v w
+    ω = x[11:13] #p q r
+    w = x[14:16] #wind-velocities
+    vw = x[17:19] #wind-velocity_dot
+
+    Q = qtoQ(q) #Rotation Matrix: [world_vector = Q * body_vector]
+    
+    ## MAIN ROTOR FORCES
+    Xmr = -u[3] * u[1] #Force along Body-X
+    Ymr = u[3] * u[2] #Force along Body-Y
+    Zmr = -u[3] #Force along Body-Z
+    
+    ## MAIN ROTOR MOMENTS
+    Lmr = Ymr * hmr #Moment about Body-X
+    Mmr = -Xmr * hmr #Moment about Body-Y
+
+    ## MAIN ROTOR YAW TORQUE AGAINST AIR
+    Qe = 0.0005 * rho * (Ωmr * Rmr)^2 * pi * Rmr^3  #(*Check Qe!)
+    
+    ## TAIL ROTOR FORCES
+    Ytr = u[4]
+    
+    ## TAIL ROTOR MOMENTS
+    Ntr = -Ytr * ltr #Moment about Body-Z
+    Ltr = Ytr * htr #Moment about Body-X
+    
+    ## DRAG FORCES
+    w_b = Q' * w #wind in Body Frame
+
+    # Note: need to take care of the sign manually...
+    Xfus = sign(w_b[1] - v[1]) * (0.5*rho*Sx_fus*Cd*(w_b[1] - v[1])*(w_b[1] - v[1])) #Drag force along Body-X
+    Yfus = sign(w_b[2] - v[2]) * (0.5*rho*Sy_fus*Cd*(w_b[2] - v[2])*(w_b[2] - v[2])) #Drag force along Body-Y
+    Zfus = sign(w_b[3] - v[3]) * (0.5*rho*Sz_fus*Cd*(w_b[3] - v[3])*(w_b[3] - v[3])) #Drag force along Body-Z
+
+    ## GRAVITY IN BODY-FRAME
+    g_matrix = Q' * [0; 0; -g]
+    
+    ## NEWTON-EULER'S EQUATIONS
+    #ṙ, q̇, v̇, ω̇ 
+    r_dot = Q * v
+    q_dot = 0.5 * L(q) * H * ω
+    v_dot = [(v[2] * ω[3]) - (v[3] * ω[2]) - g_matrix[1] + ((Xmr + Xfus) / M);
+             (v[3] * ω[1]) - (v[1] * ω[3]) + g_matrix[2] + ((Ymr + Yfus + Ytr) / M);
+             (v[1] * ω[2]) - (v[2] * ω[1]) + g_matrix[3] + ((Zmr + Zfus) / M)]
+    
+    ω_dot = [((ω[2] * ω[3] * (Iyy - Izz)) + (Lmr + Ltr)) / Ixx;
+             ((ω[1] * ω[3] * (Izz - Ixx)) + (Mmr)) / Iyy;
+             ((ω[1] * ω[2] * (Ixx - Iyy)) + (-Qe + Ntr)) / Izz]
+    
+    #return [r_dot; q_dot; v_dot; ω_dot; vw; 0; 0; 0]    
+end
+
+
+####################################################################
+#### For Visualizer: ###############################################
+####################################################################
+####################################################################
+"""
+# Animation
+function set_heli_model!(vis::Visualizer)

```

Comment:
We should just comment this out for now. We're working on another package (WIP) to hold visualizations for the models.